### DISCUSSION TITLE
ERROR: EINVAL Visual Code Studio - Thirdweb

### DISCUSSION THREAD

**Discussion by dimkalales** [pnpm/pnpm#6027] [Q&A]
Node.js is installed. I have created a new project folder in Desktop. I run the terminal and when I type:

`npx thirdweb create contract`

an error message hit me: 

Error  EINVAL: invalid argument, mkdir 'C:\Users\GRAPHI~1\AppData\Local\Temp\Graphitopia\if-you-need-to-delete-this-open-an-issue-sync-disk-cache\thirdweb:cli'
details:
{
  errno: -4071,
  syscall: 'mkdir',
  code: 'EINVAL',
  path: 'C:\\Users\\GRAPHI~1\\AppData\\Local\\Temp\\Graphitopia\\if-you-need-to-delete-this-open-an-issue-sync-disk-cache\\thirdweb:cli'
}
error stack:
• node:fs:1395 mkdirSync
    node:fs:1395:3

• index.js:74 sync
    AppData\Local\npm-cache\_npx\8d2f9c029655e602\node_modules\mkdirp\index.js:74:13

• index.js:81 sync
    AppData\Local\npm-cache\_npx\8d2f9c029655e602\node_modules\mkdirp\index.js:81:17

• index.js:205 <anonymous>
    AppData\Local\npm-cache\_npx\8d2f9c029655e602\node_modules\sync-disk-cache\index.js:205:7

• index.js:51 obj.<computed>
    AppData\Local\npm-cache\_npx\8d2f9c029655e602\node_modules\sync-disk-cache\index.js:51:19

• index.ts:86 <anonymous>
    AppData\Local\npm-cache\_npx\8d2f9c029655e602\node_modules\thirdweb\src\cli\index.ts:86:17

• task_queues:95 processTicksAndRejections
    node:internal\process\task_queues:95:5

• index.ts:73 <anonymous>
    AppData\Local\npm-cache\_npx\8d2f9c029655e602\node_modules\thirdweb\src\cli\index.ts:73:7

• command.js:935 parseAsync
    AppData\Local\npm-cache\_npx\8d2f9c029655e602\node_modules\commander\lib\command.js:935:5

• index.ts:364 k5
    AppData\Local\npm-cache\_npx\8d2f9c029655e602\node_modules\thirdweb\src\cli\index.ts:364:3

**Comment by weyert** [comment]
But you are using npx/npm?

**Reply by dimkalales** [reply]
Yes. I'm trying to create an app through Thirdweb

**Reply by iobuhov** [reply]
@dimkalales, how this relates to `pnpm`?
